### ISSUE TITLE
License Change

### ISSUE THREAD

**Issue by solverat** [dachcom-digital/pimcore-dynamic-search#93]
Dear Contributors, 

we're currently working on dual licensing this bundle (under GPLv3 and DCL [`DACHCOM Commerce License`]) to be compatible with Pimcore Enterprise and Professional. In order to do that, we need your approval (contributor license agreement). The agreement can be found here: https://github.com/dachcom-digital/pimcore-dynamic-search/blob/master/CLA.md

We therefore kindly ask the following persons to sign the CLA by **writing the following sentence as a comment under this issue**:

`I have read the CLA Document and I hereby sign the CLA`

- [x] @benwalch
- [x] @spotbot2k
- [x] @pascalmoser
- [x] @aarongerig
- [x] @DatSwissGuy
- [x] @dpfaffenbauer
- [x] @Cruiser13
- [x] @pdchaudhary
- [x] @ThisIsJustARandomGuy
- [x] @scrummer

**Comment by dpfaffenbauer** [comment]
I have read the CLA Document and I hereby sign the CLA

**Comment by pdchaudhary** [comment]
I have read the CLA Document and I hereby sign the CLA

**Comment by aarongerig** [comment]
I have read the CLA Document and I hereby sign the CLA

**Comment by scrummer** [comment]
I have read the CLA Document and I hereby sign the CLA

**Comment by Cruiser13** [comment]
I have read the CLA Document and I hereby sign the CLA

**Comment by Zodiarc** [comment]
I have read the CLA Document and I hereby sign the CLA

**Comment by drprofesq** [comment]
I'm not a contributor to this project. I think there is a bug on GitHub that associates certain types of commits with my account.

**Comment by solverat** [comment]
@drprofesq true! I've removed you from the list! Thanks anyway for the clarification!

**Comment by benwalch** [comment]
I have read the CLA Document and I hereby sign the CLA

**Comment by DatSwissGuy** [comment]
I have read the CLA Document and I hereby sign the CLA

**Comment by solverat** [comment]
Cheery ping to @spotbot2k, @ThisIsJustARandomGuy

**Comment by ThisIsJustARandomGuy** [comment]
I have read the CLA Document and I hereby sign the CLA

Am Do., 21. Nov. 2024 um 09:58 Uhr schrieb Stefan Hagspiel <
***@***.***>:

**Comment by spotbot2k** [comment]
I have read the CLA Document and I hereby sign the CLA

**Comment by solverat** [comment]
Thanks to all of you for your help! ❤️
